_posts/Durr_Hoyer_Post_1.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "A quantum algorithm for finding the minimum of an unsorted table"
+date:
+categories:
+  - Blog
+tags:
+  - qsharp
+  - algorithm
+  - open source
```

Comment:
These are tags to describe your post on the Q# community blog, I'd suggest some more descriptive ones below.

```suggestion
  - library
  - algorithm
  - in-development
```